server/src/main/java/org/apache/druid/server/coordinator/CoordinatorDynamicConfig.java

```diff
@@ -100,8 +101,12 @@ public CoordinatorDynamicConfig(
     this.killDataSourceWhitelist = parseJsonStringOrArray(killDataSourceWhitelist);
```

Comment:
Could you please rename `killDataSourceWhitelist` and `killPendingSegmentsSkipList` parameters, fields and getters to not mention "list" and the latter to mention "dataSource" (it's actual elements), while JSON property names could stay the same for compatibility.

Reply:
I'm going to change `killDataSourceWhitelist` -> `killableDatasources` and `killPendingSegmentsSkipList` -> `protectedPendingSegmentDatasources`. Makes sense?

Reply:
"DataSources". Some explanation of what do those variables mean would also be nice.

Reply:
That is actually documented in docs http://druid.io/docs/latest/configuration/index.html
![image](https://user-images.githubusercontent.com/14215045/46538303-50cfc280-c8bc-11e8-8351-85f97fd37835.png)
I could copy from there, but, I think, it would look redundantly.

Reply:
Source code should be understandable and self-contained on it's own. For somebody to understand it now, it's required to find this doc and then find how the mentioned property is mapped backed to some source code concepts, that's very difficult and time-consuming when somebody just wants to read and understand code.

Reply:
Also see #4861